bin/disco_snapshot.py

```diff
@@ -31,6 +31,8 @@ def get_parser():
     parser_create.add_argument('--size', dest='size', required=True, type=int, help='Volume size in GB')
     parser_create.add_argument('--hostclass', dest='hostclass', type=str,
                                help="hostclass that uses this snapshot")
+    parser_create.add_argument('--unencrypted', action='store_false',
```

Comment:
The name might be a little misleading.  The use of `--unencrypted` would be similar to something like `--encryption=False`. Method `create_ebs_snapshot` takes a hostclass, the size, and encryption True/False, so for an unencrypted volume we have to pass False.